Pipfile

```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 hera = { editable = true, path = "." }
+pytz = "*"
```

Comment:
There's no need to include it in the `Pipfile` as it is automatically brought in, and locked, through the `setup.py` specification 🙂  
```suggestion
```